.eslintrc

```diff
@@ -7,12 +7,12 @@
     "react/jsx-filename-extension": 0,
     "react/jsx-no-bind": 0,
     "react/jsx-uses-react": 1,
-    "react/prefer-stateless-function": 0
+    "react/prefer-stateless-function": 0,
+    "import/no-named-as-default": 0,
+    "import/no-named-as-default-member": 0,
+    "import/named": 0
   },
   "parserOptions": {
-    "ecmaVersion": 6,
-    "ecmaFeatures": {
-      "experimentalObjectRestSpread": true
-    }
+    "ecmaVersion": 6
```

Comment:
You should be good to kick this all the way up to ecmaVersion 8 or whatever the latest is